notes.txt

```diff
@@ -19,3 +19,5 @@ http://docs.opscode.com/community_plugin_knife.html
 Data bags:
 https://wiki.opscode.com/display/ChefCN/Data+Bags
 http://gettingstartedwithchef.com/more-than-one-website.html
+brk ADDED LINE IN A CAPTIAL CASE
```

Comment:
@bkrupnik This line is not all upper case. Please change this to be:

```
BRK ADDED LINE IN CAPITAL CASE
```

By having the `BRK` in upper case, it is more consistent.